Makefile

```
@@ -0,0 +1,172 @@
+# Useful shortcuts -------------------------------------------------------------
+SRCS = $(notdir $(shell ls $(SRC_DIR)/*.cc ))
+SRXS = $(notdir $(shell ls $(SRC_DIR)/*.cxx ))
+OBJS = $(SRCS:.cc=.o) 
+OBJS += $(SRXS:.cxx=.o)
+PROGS = $(notdir $(wildcard ${PROG_DIR}/*.cpp)) 
+EXES = $(PROGS:.cpp=)
+SCRIPTS = $(notdir $(wildcard ${SCRIPTS_DIR}/*.py)) 
+PYLIB_DIR = $(LIB_DIR)/python
+
+#Makefile Rules ---------------------------------------------------------------
+.PHONY: clean exe python
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)